cadence/templates/server-service-monitor.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
     app.kubernetes.io/component: {{ $service }}
     app.kubernetes.io/part-of: {{ $.Chart.Name }}
     {{- with (default $.Values.server.metrics.serviceMonitor.additionalLabels $serviceValues.metrics.serviceMonitor.additionalLabels) }}
-    {{- toYaml . | indent 4 }}
+    {{ toYaml . }}
```

Comment:
```suggestion
    {{- toYaml . | nindent 4}}
```

Reply:
It seems only remove the prefix whitespace was not right when the `additionalLabels` has more than one label, like:
```yaml
additionalLabels:
    release: prometheus
    test_label: test_label
```
the template result would be:
```yaml
apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
metadata:
  name: cadence-server-worker
  labels:
    app.kubernetes.io/name: cadence
    helm.sh/chart: cadence-0.21.1
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/instance: cadence-server
    app.kubernetes.io/version: 0.21.3
    app.kubernetes.io/component: worker
    app.kubernetes.io/part-of: cadence
    release: prometheus
test_label: test_label
spec:
   ....
```

and expected template was:

```yaml
apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
metadata:
  name: cadence-server-worker
  labels:
    app.kubernetes.io/name: cadence
    helm.sh/chart: cadence-0.21.1
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/instance: cadence-server
    app.kubernetes.io/version: 0.21.3
    app.kubernetes.io/component: worker
    app.kubernetes.io/part-of: cadence
    release: prometheus
    test_label: test_label
spec:
   ....
```

Reply:
Oh, you are right, because there is only one indentation in the beginning, I need to remove the indentation from the line and add the `| indent 4` call back to solution.

Apologies for not testing with multiple values.
The final solution should be `{{ . | nindent 4 }}`

Let me retest it quickly and fix it in a commit.

Reply:
Fixed in https://github.com/banzaicloud/banzai-charts/pull/1292/commits/0b154b2ccafe3177dacc509ada0bc384380371ef.

The right solution was (compared to the original code) to replace `indent` with `nindent`.

Reply:
@pregnor  Very very much appreciate your excellent work, it makes the deployment of cadence really simple.